src/util/utils.ts

```diff
@@ -209,7 +209,7 @@ export const isReceivedTransaction = (edgeTransaction: EdgeTransaction): boolean
 }
 
 export const isSentTransaction = (edgeTransaction: TransactionListTx | EdgeTransaction): boolean => {
-  return !!edgeTransaction.nativeAmount && edgeTransaction.nativeAmount.charAt(0) === '-'
+  return edgeTransaction.ourReceiveAddresses.length === 0 || (!!edgeTransaction.nativeAmount && edgeTransaction.nativeAmount.charAt(0) === '-')
 }
```

Comment:
The transaction details scene has its own direction logic, and there may be other places as well. That makes this change incomplete. What does it mean to receive a negative transaction, anyhow? We may need a different solution for this, like adding an `isSend: boolean` flag to `EdgeTransaction`.

Reply:
Honestly, I didn't love my solution and am definitely open to starting a conversation. The case here is a valid transaction has a zero amount and is a send but obviously doesn't start with a '-'. `ourReceiveAddresses` is the closest we have to an implied `isSend` flag which is why I opted for that rather than assuming zero amount txs should be sends.

Reply:
Ok, we can merge this as-is, and then fix the `isSend` thing in the next sprint cycle.